examples/1_simple_runbooks/configure.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+"""
+In this example we write a CLI tool with brigade and click to deploy configuration.
+"""
+from brigade.core import Brigade
+from brigade.core.configuration import Config
+from brigade.plugins.inventory.simple import SimpleInventory
+from brigade.plugins.tasks import data, networking, text
+
+
+def configure(task):
+    r = task.run(text.template_file,
+                 template="base.j2",
+                 path="../templates/{brigade_nos}")
+    task.host["config"] = r.result
+
+    r = task.run(data.load_yaml,
```

Comment:
It would be useful to add one-line comments on what you are doing in some of these steps.

Reply:
I think we need to be clear on some of this string formatting `magic`. What happens; how does it work; which variables are supported and why.

Reply:
yeah, that's explained in the tutorial :P might need a longer explanation though

Reply:
It's mentioned in the note "Note you can format strings using host data.", but I agree that it would deserve a longer explanation.